### PR TITLE
Logout refactor

### PR DESCRIPTION
**Pull request by Gwillison415**
npm package bump to 1.0.23

### CODE REVIEW

File: .env.production [hunk 1]
```diff
@@ -1 +0,0 @@
-REACT_APP_GITHUB_API_URL=https://databraid-github-api-lb-856903072.us-west-1.elb.amazonaws.coma
```
Comment (HamidAghdaee) [inline .env.production]: Why remove this?

File: src/actions/renderActions.js [hunk 2]
```diff
@@ -1,35 +1,36 @@
```
Comment (HamidAghdaee) [inline src/actions/renderActions.js:1]: What I was suggesting to you guys yesterday is more something along the lines of 
export const SET_VIEW = 'SET_VIEW';

and then one function setView which provides all the info necessary for setting the view in the action.

Comment (HamidAghdaee) [reply src/actions/renderActions.js:1]: but you don't need to address this in this PR.

File: .env.production (hunk 1 above)
Comment (Gwillison415) [reply .env.production]: there is absolutely no need to remove this and i didn't touch it. apparently i'm continuing to see unicorns- somethiing isn't working with ```git pull upstream dev```

Comment (Gwillison415) [reply .env.production]: good catch @HamidAghdaee

Comment (HamidAghdaee) [reply .env.production]: I'm not sure @Gwillison415 but I fixed this for you. You should do `git fetch upstream` and then `git rebase upstream/dev` from now on.

**Comment by HamidAghdaee** [comment]
@Gwillison415 does this work in the dashboard-spa? why isn't the version updated?

**Pull request merged**